src/app/app-interactive-map/app-interactive-map.component.ts

```diff
@@ -40,4 +47,29 @@ export class AppInteractiveMapComponent implements AfterViewInit {
       escherSettings,
     );
   }
+
+  // tslint:disable-next-line:no-any
+  handleKnockout(args: any): void {
```

Comment:
Please try to define the types (you can use the ones that are in types/escher if they are of any help)

Reply:
well, I was thinking about decide the types with you, that is why I just put any, because I was thinking about it and maybe for example for the knockout we just need the id, and for the change bounds, we need the id of the reaction and the bounds or something like that, I think not in all the function we have to have the same args. What do you think?

Reply:
it's managed by you in escher, so I leave it up to you. But yes, the id for knockout and id + the bounds sounds like rational signatures for these functions. You can maybe fill those in escher and simply log them in these callbacks?